packages/frint-compat/src/render.spec.js

```diff
@@ -45,16 +43,17 @@ describe('frint-compat › render', function () {
     expect(appStub.beforeMount).to.be.callCount(1);
   });
 
-  it('calls React.createClass to create a WrapperComponent with lifecyle methods', () => {
-    expect(React.createClass)
-      .to.be.callCount(1)
-      .and.to.be.calledWith({
-        componentDidMount: sinon.match.func,
-        componentWillUnmount: sinon.match.func,
-        displayName: 'WrapperComponent',
-        render: sinon.match.func
-      });
-  });
+  // NOTE: commented because of upgrade to React v15
```

Comment:
we should be removing the `frint-compat` package entirely instead 😛 

but I mostly keep old code lying there (usually with a `NOTE: ...` comment) so that compat can tell us if there is anything that's not compatible any more.